tests/testthat/_snaps/verb-arrange.md

```diff
@@ -37,10 +37,6 @@
       ORDER BY `a`
     Code
       lf %>% arrange(a) %>% select(-a) %>% arrange(b)
-    Condition
```

Comment:
Why does this warning go away? Because there is an arrange that's eventually applied?

Reply:
It goes away because there is no subquery anymore. It is still shown when there actually is a subquery, e.g. 

``` r
lf <- lazy_frame(a = 1, b = 1)
lf |> 
  arrange(a) |> 
  mutate(c = cumsum(b)) |> 
  filter(c == 1) |> 
  arrange(b)
#> Warning: ORDER BY is ignored in subqueries without LIMIT
#> ℹ Do you need to move arrange() later in the pipeline or use window_order() instead?
#> <SQL>
#> SELECT *
#> FROM (
#>   SELECT *, SUM(`b`) OVER (ORDER BY `a` ROWS UNBOUNDED PRECEDING) AS `c`
#>   FROM `df`
#> ) `q01`
#> WHERE (`c` = 1.0)
#> ORDER BY `b`
```

<sup>Created on 2023-01-03 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>